### PR TITLE
[CSV node] Add support for parsing empty strings and null values

### PR DESCRIPTION
**Pull request by tmdoit**
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### Database replication use case 

**Current databases state**

Database 1
name | e-mail | Language | Country
--- | --- | --- | ---
tmdoit | tmdoit@doit.pl | PL | Poland

Database 2
name | e-mail | Language | Country
--- | --- | --- | ---
tmdoit | tmdoit@doit.pl | PL | Poland

**After "Database 1" update**

Database 1
name | e-mail | Language | Country
--- | --- | --- | ---
tmdoit | "" | NULL | NULL

Database 2
name | e-mail | Language | Country
--- | --- | --- | ---
tmdoit | tmdoit@doit.pl | PL | Poland

To reflect changes in "Database 2" I use CSV string: '§,"",,' where:
- §: means field doesn't changed
- "": means field is set to empty string
- ,,: means field is set to NULL value
- ,: means field is set to NULL value

Now using CSV node it's impossible to get null values and empty strings - in both cases there is no value returned.

## Checklist

- [ ] I have read the [contribution guidelines](https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md)
- [x ] For non-bugfix PRs, I have discussed this change on the forum/slack team.
- [ ] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

File: packages/node_modules/@node-red/nodes/locales/en-US/messages.json [hunk 1]
```diff
@@ -698,7 +698,9 @@
             "output": "Output",
             "includerow": "include column name row",
             "newline": "Newline",
-            "usestrings": "parse numerical values"
+            "usestrings": "parse numerical values",
+            "parse_empty_strings": "parse empty strings",
```
Comment (dceejay) [inline packages/node_modules/@node-red/nodes/locales/en-US/messages.json]: not sure parse is the right word here - maybe just "include empty strings" ?

File: packages/node_modules/@node-red/nodes/locales/en-US/messages.json [hunk 2]
```diff
@@ -698,7 +698,9 @@
...
+            "parse_null_values": "parse null values"
```
Comment (dceejay) [inline packages/node_modules/@node-red/nodes/locales/en-US/messages.json]: Likewise "include null values" - or is it null columns ?

File: packages/node_modules/@node-red/nodes/locales/en-US/messages.json (hunk 1 above)
Comment (tmdoit) [reply packages/node_modules/@node-red/nodes/locales/en-US/messages.json]: Done.

File: packages/node_modules/@node-red/nodes/locales/en-US/messages.json (hunk 2 above)
Comment (tmdoit) [reply packages/node_modules/@node-red/nodes/locales/en-US/messages.json]: Done.

**Comment by dceejay** [comment]
Thanks - there was no need to change all the variable names - just the labels/translations - but thanks !

**Pull request merged**